sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/RangeSpec.java

```diff
@@ -0,0 +1,41 @@
+package com.mesosphere.sdk.specification;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+
+/**
+ * Representation of a port range, consisting of a begin and an end.
```

Comment:
May be update the javadoc here to clarify on the nature of (open vs closed on left and right) interval ?!

Reply:
please state if inclusive in the docs